src/Bonnier/BCM/Services/MetaTag.php

```diff
@@ -0,0 +1,198 @@
+<?php
+/**
+ * MetaTag class file
+ */
+namespace Bonnier\BCM\Services;
+
+use Bonnier\BCM\Settings\SettingsPage;
+
+/**
+ * MetaTag class
+ */
+class MetaTag {
+	
+	/**
+	 * Settings object
+	 *
+	 * @var SettingsPage $objSettings
+	 */
+	private static $objSettings;
+
+	/**
+	 * Bootstrap the service by adding to the wordpress action wp_head
+	 *
+	 * @param SettingsPage $objSettings
+	 * @return null
+	 */
+	public static function bootstrap(SettingsPage $objSettings) {
+		self::$objSettings = $objSettings;
+		add_action('wp_head', [__CLASS__, 'add_head_tags']);
+	}
+
+	/**
+	 * Add meta tags
+	 *
+	 * @return null
+	 */
+	public static function add_head_tags() {
+		
+		if (self::$objSettings->enabled) {
+			
+			// add mandatory tags
+			$arrTags = [
+				'bcm-brand' => self::$objSettings->brand,
+				'bcm-country' => self::$objSettings->country,
+				'bcm-type' => self::$objSettings->type
+			];
+			
+			// add subcategory tag
+			if (self::$objSettings->sub) {
+				$arrTags['bcm-sub'] = self::$objSettings->sub;
+			}
+			
+			// add tablet breaking point
+			if (self::$objSettings->tablet_breakpoint) {
+				$arrTags['bcm-tablet-breakpoint'] = self::$objSettings->tablet_breakpoint;
+			}
+			
+			// add mobile breaking point
+			if (self::$objSettings->mobile_breakpoint) {
+				$arrTags['bcm-mobile-breakpoint'] = self::$objSettings->mobile_breakpoint;
+			}
+			
+			// we only apply article tags when necessary
+			if (!is_front_page() && (is_singular() || is_single())) {
+				$arrTags = array_merge($arrTags, self::get_article_tags());
+			}
+			
+			// write the actual tags
+			self::write_meta_tags($arrTags);
+		}
+	}
+	
+	/**
+	 * Get specific article meta tags
+	 *
+	 * @return array
+	 */
+	private static function get_article_tags() {
+		$arrArticleTags = [
+			'bcm-title' => self::get_bcm_title()
+		];
+
+		// set content type
+		if ($strContentType = self::get_bcm_content_type()) {
+			$arrArticleTags['bcm-content-type'] = $strContentType;
+		}
+		
+		// set all categories
+		if ($arrCategories = self::get_bcm_categories()) {
+			$arrArticleTags['bcm-categories'] = implode(',', $arrCategories);
+		}
+		
+		// set all tags
+		if ($arrTags = self::get_bcm_tags()) {
+			$arrArticleTags['bcm-tags'] = implode(',', $arrTags);
+		}
+		
+		// set advertorial type
+		if ($strAdvertorialType = self::get_bcm_advertorial_type()) {
+			$arrArticleTags['bcm-advertorial-type'] = implode(',', $strAdvertorialType);
```

Comment:
Wouldn't this be a string and thus you should just print the value rather than imploding it?